Sources/ConcurrencyHelpers/Protected.swift

```diff
@@ -0,0 +1,36 @@
+/// Thread safe access to simple variable protected by lock
+@propertyWrapper
+public final class Protected<T> {
+    var value: T
+    var lock: ConcurrencyHelpers.Lock
```

Comment:
```suggestion
    var lock = ConcurrencyHelpers.Lock()
```
And can drop one line from `init()`?

Reply:
Fixed.